cvat/apps/engine/mixins.py

```diff
@@ -474,14 +450,30 @@ def export_dataset_v1(
         },
     )
     @action(detail=True, methods=['POST'], serializer_class=None, url_path='dataset/export')
-    def export_dataset_v2(self, request: ExtendedRequest, pk: int):
+    def initiate_dataset_export(self, request: ExtendedRequest, pk: int):
         self._object = self.get_object() # force call of check_object_permissions()
 
-        save_images = is_dataset_export(request)
-        callback = self.get_export_callback(save_images)
+        export_manager = DatasetExportManager(self._object, request)
+        export_manager.initialize_export_args()
+
+        return export_manager.export()
 
-        dataset_export_manager = DatasetExportManager(self._object, request, callback, save_images=save_images, version=2)
-        return dataset_export_manager.export()
+    @extend_schema(summary='Download a prepared dataset file',
+        parameters=[
+            OpenApiParameter('rq_id', description='Request ID',
+                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=True),
+        ],
+        responses={
+            '200': OpenApiResponse(description='Download of file started'),
+            '204': OpenApiResponse(description='No prepared dataset file related with provider request ID'),
+        },
+        exclude=True, # private API endpoint that should be used only as result_url
```

Comment:
Given this, does it make sense to set any other parameters? Seems like they're just going to be ignored.

Reply:
If I understand you correctly, you are talking about defining the whole API endpoint schema here, right? I think it would still be better to explicitly define the endpoint schema because it would be helpful for developers.

Reply:
Okay, suit yourself.

IMO, since you don't need to provide a full OpenAPI schema, it would be more readable to summarize the behavior of the endpoint in a concise freeform comment (you could even reduce duplication by putting that comment into `download_file`). But you're more likely to edit this part of the code than me, so I'll defer to your judgement.